entity-framework/core/modeling/query-types.md

```diff
@@ -15,26 +15,27 @@ Query Types are read-only query result types that can be added to the EF Core mo
 
 They are conceptually similar to Entity Types in that:
 
-- They are POCO C# types that are added to the model, either in ```OnModelCreating``` using the ```ModelBuilder.Query``` method, or via a DbContext "set" property (for query types such a property is typed as ```DbQuery<T>``` rather than ```DbSet<T>```).
-- They support much of the same mapping capabilities as regular entity types. For example, inheritance mapping, navigations (see limitiations below) and, on relational stores, the ability to configure the target database schema objects via ```ToTable```, ```HasColumn``` fluent-api methods (or data annotations).
+- They are simple classes that are added to the model, either in `OnModelCreating` using the `ModelBuilder.Query` method, or via a DbContext "set" property (for query types such a property is typed as `DbQuery<T>` rather than `DbSet<T>`).
```

Comment:
What does "simple class" mean? Are we moving away from talking about "POCOs"?

Reply:
I am happy that our customers can take "persistence ignorance" for granted (at least to the degree that our mapping is flexible and keeps become more flexible). 

I took "POCO" out because I believe that:
- For too many customers it conveys a class with a parameterless constructor and public automatic properties, and I refuse to feed that misconception :smile: 
- There are also customers out there who missed the old debate and for which the term means nothing.

It took "C# type" because I believe .NET class is a much simpler and slightly more accurate way to refer to what it is, but .NET is also redundant so I avoided it.

I put simple there trying to preserve some of the meaning, but now that I think more about it, I am inclined to remove it. If I had to qualify "class" with anything, I would probalby use "data class". What do you think?